projects/demo/src/components/control/index.html

```diff
@@ -0,0 +1,36 @@
+<tr>
+    <td colspan="3">
+        <span tuiTitle>
+            <strong>ControlOptions</strong>
+            <span tuiSubtitle>Applied as a host directive</span>
+        </span>
+    </td>
+</tr>
+
+<tr
+    *ngIf="!hiddenOptions.includes('readonly')"
```

Comment:
I think it's `readOnly` with capital `O`

Reply:
Renamed